rules/windows/evasion_suspicious_wmi_script.toml

```
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "development"
```

just confirming this is not meant for prod

---

This _was_ intentional, unless we think the wildcard performance concerns are negligible

---

I'll bump it and see how it works out

```suggestion
maturity = "production"
```